bulbs/special_coverage/models.py

```diff
@@ -104,11 +104,21 @@ def _save_percolator(self):
             return
 
         # We'll need this data, to decide which special coverage section to use
+        q["sponsored"] = bool(self.campaign)
         if self.campaign:
-            q["sponsored"] = True
-            q["start_date"] = self.campaign.start_date
-            q["end_date"] = self.campaign.end_date
-
+            # ES v1.4 percolator doesn't support DateTime ranges, use integer epoch seconds instead
+            q["start_date"] = datetime_to_epoch_seconds(self.campaign.start_date)
```

Comment:
Or should these fields be renamed to "start_date_epoch" to make their integer type more explicit? Downside is more fields and need to add ES mapping